### PR TITLE
Fix #8, Fix #9 — mention other compilers, distributions, Unicode

### LINKED ISSUE #8: TeXstudio is a nicer editor
TeXstudio is a fork of TeXmaker with more features. Perhaps you should promote that instead?

### LINKED ISSUE #9: On LaTeX compilers and Unicode
> Some languages won't work right out of the box. You will need to include some packages for the font to render. Also, you will learn about "packages" later. For example:

``` tex
\documentclass[a4paper]{article}

\usepackage[T5]{fontenc}
\usepackage[utf8]{inputenc}

\begin{document}

Xin chào thế giới. This is Hellow World in Vietnamese.

\end{document}
```

It looks like you’re still living in the 90s where Unicode support was neglected. You see, two projects have solved this problem: XeTeX and LuaTeX. Those are two compilers with Unicode support.

``` tex
\documentclass[a4paper]{article}
\usepackage{fontspec}
\usepackage{polyglossia}
\setmainfont[]{Baskerville} % any TTF or OTF fonts from my system will do
\begin{document}
Xin chào thế giới. This is Hello World in Vietnamese.

The quick brown fox jumps over the lazy dog.

Pchnąć w tę łódź jeża lub ośm skrzyń fig.

Quiere la boca exhausta vid, kiwi, piña y fugaz jamón.

Vogt Nyx: »Büß du ja zwölf Qirsch, Kämpe!«

Любя, съешь щипцы, — вздохнёт мэр, — кайф жгуч.
\end{document}
```

International TeX with Baskerville

If I don’t set any font (remove line with comment), the font will default to Latin Modern, which is a reworking of Computer Modern with support for many Latin diacritic characters. The Russian part fails.

International-ish TeX with Latin Modern

In TeXstudio (and probably TeXMaker): change Build → Default Compiler in the settings. I’m using LuaLaTeX.

### PR DESCRIPTION
**Pull request by Kwpolska**
Fix #8, Fix #9 — mention other compilers, distributions, Unicode.

### CODE REVIEW

**Comment by luong-komorebi** [comment]
Thank you 💃

File: readme.md [hunk 1]
```diff
@@ -91,18 +98,14 @@ Xin chào thế giới. This is Hello World in Vietnamese.
...
 ```
 
-Here we use two packages `usepackage[T5]{fontenc}` and `usepackage[utf8]{inputenc}` . This is really simple to understand as the package will import font encoders to display your content correctly. If you are using TexMaker this is what the above code display :
-
-![](http://i.imgur.com/OUSPekM.png)
-
-vs without the packages :package::  
-
-![](http://i.imgur.com/Upd47xH.png)  
+The default font (Latin Modern) does not support all characters. You can,
```
Comment (vermiculus) [inline readme.md:101]: The default font is Computer Modern, not Latin Modern. Latin Modern is used when `lmodern` is loaded as a package.

File: readme.md [hunk 2]
```diff
@@ -75,14 +77,19 @@ It should look like this in TexMaker:
 * A document is wrapped by the **\begin{document}** and **\end{document}** . Think of this as the heart of the document, as the `main()` in *java* or *C++* ... without which the document can't be rendered.
 * The part between begin and end ( which, in this case, is `Hello World` ) is simply your own content.
 
-### :warning: Important :warning:   
-**Some languages won't work right out of the box. You will need to include some packages for the font to render. Also, you will learn about "packages" later.** For example:  
+## Multilingual usage
+
+To use TeX with other languages, you some options. If you use pdfLaTeX, the
```
Comment (vermiculus) [inline readme.md:82]: 'you have some options'.

File: readme.md (hunk 1 above)
Comment (Kwpolska) [reply readme.md:101]: Incorrect.

```
[kwpolska@Kw-Farore /tmp]% lualatex hello.tex
This is LuaTeX, Version 0.95.0 (TeX Live 2016)
 restricted system commands enabled.
(./hello.tex
LaTeX2e <2016/03/31> patch level 3
Babel <3.9r> and hyphenation patterns for 1 language(s) loaded.
(/usr/local/texlive/2016/texmf-dist/tex/latex/base/article.cls
Document Class: article 2014/09/29 v1.4h Standard LaTeX document class
(/usr/local/texlive/2016/texmf-dist/tex/latex/base/size10.clo))
(/usr/local/texlive/2016/texmf-dist/tex/latex/fontspec/fontspec.sty
(/usr/local/texlive/2016/texmf-dist/tex/latex/l3kernel/expl3.sty
(/usr/local/texlive/2016/texmf-dist/tex/latex/l3kernel/expl3-code.tex)
(/usr/local/texlive/2016/texmf-dist/tex/latex/l3kernel/l3pdfmode.def))
(/usr/local/texlive/2016/texmf-dist/tex/latex/l3packages/xparse/xparse.sty)
(/usr/local/texlive/2016/texmf-dist/tex/luatex/luaotfload/luaotfload.sty(using
write cache: /usr/local/texlive/2016/texmf-var/luatex-cache/generic)(using read
 cache: /usr/local/texlive/2016/texmf-var/luatex-cache/generic /Users/kwpolska/
Library/texlive/2016/texmf-var/luatex-cache/generic)
luaotfload | main : initialization completed in 0.180 seconds)
(/usr/local/texlive/2016/texmf-dist/tex/latex/fontspec/fontspec-luatex.sty
(/usr/local/texlive/2016/texmf-dist/tex/latex/base/fontenc.sty
(/usr/local/texlive/2016/texmf-dist/tex/latex/euenc/eu2enc.def)
(/usr/local/texlive/2016/texmf-dist/tex/latex/euenc/eu2lmr.fd)(compiling luc: /
usr/local/texlive/2016/texmf-var/luatex-cache/generic/fonts/otl/lmroman10-regul
ar.luc)(load luc: /Users/kwpolska/Library/texlive/2016/texmf-var/luatex-cache/g
eneric/fonts/otl/lmroman10-regular.luc))
(/usr/local/texlive/2016/texmf-dist/tex/xelatex/xunicode/xunicode.sty
(/usr/local/texlive/2016/texmf-dist/tex/latex/tipa/t3enc.def(compiling luc: /us
r/local/texlive/2016/texmf-var/luatex-cache/generic/fonts/otl/lmromanslant10-re
gular.luc)(load luc: /Users/kwpolska/Library/texlive/2016/texmf-var/luatex-cach
e/generic/fonts/otl/lmromanslant10-regular.luc)(compiling luc: /usr/local/texli
ve/2016/texmf-var/luatex-cache/generic/fonts/otl/lmroman10-italic.luc)(load luc
: /Users/kwpolska/Library/texlive/2016/texmf-var/luatex-cache/generic/fonts/otl
/lmroman10-italic.luc)(compiling luc: /usr/local/texlive/2016/texmf-var/luatex-
cache/generic/fonts/otl/lmroman10-bold.luc)(load luc: /Users/kwpolska/Library/t
exlive/2016/texmf-var/luatex-cache/generic/fonts/otl/lmroman10-bold.luc)
(/usr/local/texlive/2016/texmf-dist/tex/latex/euenc/eu2lmss.fd)(compiling luc:
/usr/local/texlive/2016/texmf-var/luatex-cache/generic/fonts/otl/lmsans10-regul
ar.luc)(load luc: /Users/kwpolska/Library/texlive/2016/texmf-var/luatex-cache/g
eneric/fonts/otl/lmsans10-regular.luc))
(/usr/local/texlive/2016/texmf-dist/tex/latex/graphics/graphicx.sty
(/usr/local/texlive/2016/texmf-dist/tex/latex/graphics/keyval.sty)
(/usr/local/texlive/2016/texmf-dist/tex/latex/graphics/graphics.sty
(/usr/local/texlive/2016/texmf-dist/tex/latex/graphics/trig.sty)
(/usr/local/texlive/2016/texmf-dist/tex/latex/graphics-cfg/graphics.cfg)
(/usr/local/texlive/2016/texmf-dist/tex/latex/graphics-def/luatex.def
(/usr/local/texlive/2016/texmf-dist/tex/generic/oberdiek/infwarerr.sty)
(/usr/local/texlive/2016/texmf-dist/tex/generic/oberdiek/ltxcmds.sty)))))
(/usr/local/texlive/2016/texmf-dist/tex/latex/fontspec/fontspec.cfg)))
(/usr/local/texlive/2016/texmf-dist/tex/latex/polyglossia/polyglossia.sty
(/usr/local/texlive/2016/texmf-dist/tex/latex/etoolbox/etoolbox.sty)
(/usr/local/texlive/2016/texmf-dist/tex/latex/makecmds/makecmds.sty)
(/usr/local/texlive/2016/texmf-dist/tex/latex/xkeyval/xkeyval.sty
(/usr/local/texlive/2016/texmf-dist/tex/generic/xkeyval/xkeyval.tex
(/usr/local/texlive/2016/texmf-dist/tex/generic/xkeyval/xkvutils.tex)))
(/usr/local/texlive/2016/texmf-dist/tex/generic/oberdiek/ifluatex.sty)
(/usr/local/texlive/2016/texmf-dist/tex/generic/ifxetex/ifxetex.sty)
(/usr/local/texlive/2016/texmf-dist/tex/luatex/luatexbase/luatexbase.sty
(/usr/local/texlive/2016/texmf-dist/tex/luatex/ctablestack/ctablestack.sty)))
(./hello.aux) (/usr/local/texlive/2016/texmf-dist/tex/latex/tipa/t3cmr.fd)
(/usr/local/texlive/2016/texmf-dist/tex/context/base/mkii/supp-pdf.mkii
[Loading MPS to PDF converter (version 2006.09.02).]
) (/usr/local/texlive/2016/texmf-dist/tex/latex/oberdiek/epstopdf-base.sty
(/usr/local/texlive/2016/texmf-dist/tex/latex/oberdiek/grfext.sty
(/usr/local/texlive/2016/texmf-dist/tex/generic/oberdiek/kvdefinekeys.sty))
(/usr/local/texlive/2016/texmf-dist/tex/latex/oberdiek/kvoptions.sty
(/usr/local/texlive/2016/texmf-dist/tex/generic/oberdiek/kvsetkeys.sty
(/usr/local/texlive/2016/texmf-dist/tex/generic/oberdiek/etexcmds.sty)))
(/usr/local/texlive/2016/texmf-dist/tex/generic/oberdiek/pdftexcmds.sty
(/usr/local/texlive/2016/texmf-dist/tex/generic/oberdiek/ifpdf.sty)
(/usr/local/texlive/2016/texmf-dist/tex/generic/oberdiek/luatex-loader.sty
(/usr/local/texlive/2016/texmf-dist/scripts/oberdiek/oberdiek.luatex.lua)))
(/usr/local/texlive/2016/texmf-dist/tex/latex/latexconfig/epstopdf-sys.cfg))
[1{/usr/local/texlive/2016/texmf-var/fonts/map/pdftex/updmap/pdftex.map}]
(./hello.aux))
 396 words of node memory still in use:
   3 hlist, 1 vlist, 1 rule, 7 glue, 4 attribute, 44 glue_spec, 4 attribute_lis
t, 1 write nodes
   avail lists: 2:13,3:4,4:2,5:12,6:47,7:31,8:1,9:6,11:1
</Users/kwpolska/Library/Fonts/lmroman10-regular.otf>
Output written on hello.pdf (1 page, 5062 bytes).
Transcript written on hello.log.
```

Comment (Kwpolska) [reply readme.md:101]: (This is the output of my polyglossia/fontspec example, without `\usepackage{lmodern}`. That package is useless in Xe/LuaTeX anyway.

Comment (luong-komorebi) [reply readme.md:101]: I am not professional in this so I am not gonna join the discussion here. I respect both of your opinions.  
One thing I'd like to mention is that here the repo focuses on simplicty and accessibility for majority of new-to-LaTeX people. Some problems may bother experts, some may not and are acceptable for newcomers. Think carefully 👍

**Pull request merged**